<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customerReviews</class>
 <widget class="QWidget" name="customerReviews">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Resources/night.jpg);</string>
  </property>
  <widget class="QPushButton" name="write">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>520</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
font: 15pt &quot;American Typewriter&quot;;
color: white;</string>
   </property>
   <property name="text">
    <string>Submit a Testimonial</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="custTest">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>221</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;American Typewriter&quot;;
color: white;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="submitForm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>251</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;American Typewriter&quot;;
color: white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Zapfino&quot;;
color: white;</string>
   </property>
   <property name="text">
    <string>Feel free to write a review below...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLine">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;American Typewriter&quot;;
color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:
rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Please rate us: </string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>158</width>
     <height>77</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="veryInterested">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Very interested</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="somewhatInterested">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Somewhat interested</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="notInterested">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Not interested</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="neverCallAgain">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Never will call again</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
